contrib/vagrant/provision-config.sh

```diff
@@ -12,11 +15,63 @@ ORIGIN_ROOT=$(
 )
 source ${ORIGIN_ROOT}/contrib/vagrant/provision-util.sh
 
-# Passed as arguments to provisioning from Vagrantfile
+# Passed as arguments to provisioning script
 MASTER_IP=${1:-""}
-NUM_MINIONS=${2:-""}
-MINION_IPS=${3:-""}
-INSTANCE_PREFIX=${4:-${OS_INSTANCE_PREFIX:-openshift}}
+NODE_COUNT=${2:-${OPENSHIFT_NUM_MINIONS:-2}}
+NODE_IPS=${3:-""}
+INSTANCE_PREFIX=${4:-${OPENSHIFT_INSTANCE_PREFIX:-openshift}}
+
+# Set defaults for optional arguments
+FIXUP_NET_UDEV=false
+NETWORK_PLUGIN=${OPENSHIFT_NETWORK_PLUGIN:-""}
+NODE_INDEX=0
+CONFIG_ROOT=${ORIGIN_ROOT}
+SKIP_BUILD=${OPENSHIFT_SKIP_BUILD:-false}
```

Comment:
The OPENSHIFT_SKIP_BUILD stuff seems like it would be easier to use if it was a command-line option to dind-cluster.sh instead of an environment variable. You could also have separate flags for skipping the image build and skipping the binary build.